charter.html

```diff
@@ -167,9 +167,9 @@ <h2>Scope</h2>
             processing of personal information.  Ways in which new features might
             enable inappropriate processing include (but are not limited to)
             enabling
-            of <a href="https://w3ctag.github.io/privacy-principles/#hl-recognition-cross-site">cross-site
+            of <a href="https://www.w3.org/TR/2022/DNOTE-privacy-principles-20221214/#hl-recognition-cross-site">cross-site
             or cross context recognition</a> of users or
```

Comment:
Also, cross-context recognition is an explicitly-defined term now, so maybe that would be preferred, rather than 
"cross-site or cross-context":

https://www.w3.org/TR/2023/DNOTE-privacy-principles-20230223/#dfn-cross-context-recognition